static/js/zamboni/static_theme.js

```diff
@@ -0,0 +1,244 @@
+$(document).ready(function() {
+    function StaticThemeBrowserPreview() {
+        this.generate = function($div) {
+            $div.append($(
+                '<svg id="SvgjsSvg1006" width="100%" height="100%" xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs">' +
```

Comment:
Ugh. Not a fan of the inline svg, especially when it contains text that should probably be translated.

Reply:
I was generating it real time with svg.js at first but I didn't want to include the 500k js library so I copied the xml generated. 😆
I'll see what I can do with the external file - I'm not 100% how well DOM manipulation works when it's not literal xml in the html.

Reply:
https://github.com/mozilla/addons-server/pull/6993/commits/da0ad769443c0fda48457a9f857bbf89fc0e24af